tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -19,30 +19,32 @@
     "jp-MarkdownHeadingCollapsed": true
    },
    "source": [
-    "## Overview of Binary Contingency Scores\n",
+    "## Scores for Binary Categorical Forecasts\n",
     "\n",
-    "Examples of contingency scores include: hit rate (true positive rate), false alarm rate and accuracy.\n",
+    "Examples of binary categorical scores include: hit rate (true positive rate), false alarm rate and accuracy.\n",
     "\n",
     "These scores are calculated based on a binary contingency table, which comprises true positives (hits), true negatives (correct negatives), false positives (false alarms) and false negatives (misses).\n",
     "\n",
     "Producing these categorical statistics is inherently a three-step process.\n",
     "\n",
     "1. For data that is either continuous or categorical, first convert it to binary data using an \"event definition\". For data that is already binary, go straight to step two.\n",
+    " \n",
     "2. Produce a contingency manager object, which:\n",
-    "   - stores the true positives, true negativates, false positives and false negatives, and\n",
-    "   - allows the data to be aggragated and transformed (for example, reducing the latitude and longitude dimensions while preserving the height dimension in order to assess the accuracy at different vertical levels).\n",
-    "3. Calculate the desired metrics from the contingency tables produced in step two. `scores` contains the following binary congtingency scores:\n",
+    "   - stores the true positives, true negatives, false positives and false negatives, and\n",
+    "   - allows the data to be aggregated and transformed (for example, reducing the latitude and longitude dimensions while preserving the height dimension in order to assess the accuracy at different vertical levels).\n",
+    " \n",
+    "3. Calculate the desired metrics from the contingency tables produced in step two. `scores` contains the following binary categorical scores:\n",
     "   - accuracy (fraction correct)\n",
     "   - frequency bias (bias score)\n",
     "   - hit rate (true positive rate, probability of detection, sensitivity)\n",
     "   - false alarm rate (probability of false detection) - not to be confused with false alarm ratio\n",
-    "   - success ratio\n",
+    "   - success ratio \n",
     "   - threat score (critical success index)\n",
-    "   - the Peirce skill score (true skill statistic, Hanssen and Kuipers discriminant)\n",
-    "   - specificity\n",
+    "   - Peirce skill score (true skill statistic, Hanssen and Kuipers discriminant)\n",
+    "   - specificity (true negative rate)\n",
     "\n",
```

Comment:
This list will need updating (new metrics, and more additional names for existing metrics) in line with pull request #316. Not sure if that is easier being handled now, or should be done later as a separate pull request.

Reply:
Hang on - I think @nicholasloveday has already taken care of this in #314

Reply:
It will be easier to handle any discrepancies later. Once the various merges are all in, we should loop back to check the final consistency.